config/database.yml

```diff
@@ -3,6 +3,10 @@
 #
 #   Ensure the SQLite 3 gem is defined in your Gemfile
 #   gem 'sqlite3'
+<% user = ENV.key?("POSTGRESQL_ADMIN_PASSWORD") ? "root" : ENV["POSTGRESQL_USER"] %>
```

Comment:
This had me confused but I'm guessing if it sees that key then the administrator name **has** to be "root"?

Reply:
Yeah, the standard postgres container in openshift has a POSTGRESQL_ADMIN_PASSWORD environment variable that allows you to specify the admin password. This would allow you to connect as admin and configure with some weaknesses. It's also a part of Openshift's sample rails application that is likely in a ton of new configs